modules/administration-guide/partials/assembly_backups-of-persistent-volumes.adoc

```diff
@@ -0,0 +1,6 @@
+[id="backups-of-persistent-volumes_{context}"]
+= Backups of persistent volumes
+
+Persistent Volumes (PVs) store workspace data for {prod-short} similarly to how workspace data are stored for desktop IDEs on a local hard disk drive. PVs must be backed up periodically to prevent data loss. The recommended approach is to use storage-agnostic tools for backing up and restoring {platforms-name} resources, including PVs.
```

Comment:
Do you mean "Persistent Volumes (PVs) store workspace data for {prod-short} similarly to how desktop IDEs store workspace data on a local hard disk drive." OR "Persistent Volumes (PVs) store workspace data for {prod-short} on a local hard disk drive, similarly to how desktop IDEs store workspace data."
I believe PVs can be stored on a variety of storage types, not just a local drive.

Reply:
The IBM SG and SSG both ask us to avoid using "recommend." Instead, they ask us to make opinionated  statements. For example, the SSG says:
```
no recommend (verb)
Description: Avoid "recommends". Instead of "Red Hat recommends", direct users to take the recommended action. This allows Red Hat to be more prescriptive in documentation and prevent any user uncertainty, and is easier for upstream/downstream coordinated efforts.
```
The IBM SG says, among other things, the following:
```
[Red X] If you have a large data set, you are recommended to disable the automatic search.
[Green checkmark] If you have a large data set, disable the automatic search to improve performance.
```

Reply:
Please review multiple instances of "recommend*" in this PR for similar concerns.

Reply:
Thank you.